definitions/pipelines/somatic_exome.cwl

```diff
@@ -505,13 +515,22 @@ steps:
             vcf: somalier_vcf
         out:
             [somalier_pairs, somalier_samples]
+
+    pad_target_intervals:
+        run: ../tools/interval_list_expand.cwl
+        in: 
+            interval_list: target_intervals
+            roi_padding: target_interval_padding
+        out:
+            [expanded_interval_list]
+
```

Comment:
We haven't been setting steps apart with whitespace on either side like this.  I'd prefer to keep the files consistent.

Reply:
I definitely prefer some whitespace for readability, but I'll drop it for consistency's sake. (and a strong desire not to go through the repo reformatting everything!